packages/core/lib/patchers/aws_p.js

```diff
@@ -85,10 +85,17 @@ function captureAWSRequest(req) {
   }
 
   var traceId = parent.segment ? parent.segment.trace_id : parent.trace_id;
+  const data = parent.segment ? parent.segment.additionalTraceData : parent.additionalTraceData;
 
   var buildListener = function(req) {
-    req.httpRequest.headers['X-Amzn-Trace-Id'] = 'Root=' + traceId + ';Parent=' + subsegment.id +
+    let traceHeader = 'Root=' + traceId + ';Parent=' + subsegment.id +
       ';Sampled=' + (subsegment.notTraced ? '0' : '1');
+    if (data != null) {
```

Comment:
`!==` is a better alternative to `!=` in this case since it performs a more [strict type check ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_inequality)

Reply:
Does the type checking matter in JS when comparing with `null`? My understanding is that the expression `data != null` would return `false` when the `data` is either `null` or `undefined`. Whereas, `data !== null` only returns `false` when data is specifically `null`. Wouldn't that make `!=` more appropriate when comparing with `null`?

Reply:
Got it, thanks for the clarification! If the possible values we are checking for are null/undefined, then` !=` is the way to go.

Reply:
Yup. We intend to handle `undefined` and `null` the same way here.